### PR TITLE
docs: Note that the `send_transaction` RPC is asynchronous

### PR DESCRIPTION
**Pull request by eval-exec**
### What problem does this PR solve?

Follow up https://github.com/nervosnetwork/docs.nervos.org/pull/188

What's Changed:

### Related changes

- docs: note `send_transaction` RPC is asynchronous
### Check List 

Tests 

- Unit test
- Integration test
- Manual test (add detailed scripts or steps below)
- No code ci-runs-only: [ quick_checks,linters ]

Side effects

- None

### Release note 

```release-note
Title Only: Include only the PR title in the release note.
```

### CODE REVIEW

File: rpc/README.md [hunk 1]
```diff
@@ -4343,6 +4343,8 @@ RPC Module Pool for transaction memory pool.
 
 Submits a new transaction into the transaction pool. If the transaction is already in the pool, rebroadcast it to peers.
 
+It should be noted that `send_transaction` is asynchronous. That is, the return of the transaction hash, does NOT mean that the transaction is fully verified. If you need to follow up on the status of the transaction, it is recommended to use the `get_transaction` rpc.
```
Comment (chenyukang) [inline rpc/README.md:4346]: Please note `send_transaction` is asynchronous.

Comment (chenyukang) [reply rpc/README.md:4346]: The return value of the request does NOT mean the transaction is fully verified. If you want to follow up on the status of the transaction, please use the `get_transaction` rpc to query the latest status.

**Review by driftluo — APPROVED** [review]

File: rpc/README.md (hunk 1 above)
Comment (eval-exec) [reply rpc/README.md:4346]: I ask ChatGPT to reword this. It give me:
```
    /// Please note that `send_transaction` is an asynchronous process.
    /// The return of `send_transaction does NOT indicate that the transaction have been fully verified.
    /// If you want to track the status of the transaction, please use the `get_transaction`rpc.

```

Comment (chenyukang) [reply rpc/README.md:4346]: Simple and short words is a better style.

**Review by chenyukang — APPROVED** [review]

**Review by zhangsoledad — APPROVED** [review]

**Pull request merged**